content/pages/sprint/2021/08early-pycht.rst

```diff
@@ -0,0 +1,79 @@
+=================================================
+Scisprint 2021 Early August: Python ZH TW Doc
+=================================================
+
+:date: 2021-06-29 15:24
+:url: sprint/2021/08early-pycht
+:save_as: sprint/2021/08early-pycht.html
+
+Scisprint 2021 Early August: Python ZH TW Doc
+=================================================
+
+This is the second sprint for for translating `Python official
+document <https://docs.python.org/3/>`__ from English to traditional Chinese
+(Taiwan; ZH TW locale).
+
+The agenda:
```

Comment:
I am not sure we should use the same agenda as the previous sprint (https://sciwork.dev/sprint/2021/07early-pycht).  Perhaps it is good to have it updated?

Reply:
Yeah. I've added and updated the description about rescheduling the going of the project and the review workflow. But I also keep some same topics that we have't discussed too much, I guess we could have more discussion about that in the next sprint.

Reply:
Sorry I overlooked the updates already made.